pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -96,6 +96,9 @@ type ControllerConfigSpec struct {
 	// Nobody is also changing this once the cluster is up and running the first time, so, disallow
 	// regeneration if this changes.
 	NetworkType string `json:"networkType,omitempty"`
+
+	// ReleaseImage is the image used for cluster installation
+	ReleaseImage string `json:"releaseImage"`
```

Comment:
since we're not setting this - nor using it, I think the changes in the bootstrap file are what we need till we'll be able to play with the image name itlsef